server/src/main/java/com/broadcom/lsp/cobol/core/messages/KeyNotFoundException.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2020 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+package com.broadcom.lsp.cobol.core.messages;
+
+import lombok.NonNull;
+
+public class KeyNotFoundException extends RuntimeException {
+  public KeyNotFoundException(@NonNull String s) {}
```

Comment:
Shouldn't there be a call for `RuntimeException` constructor?

Reply:
Yes, Thanks for pointing that out.